README.md

```diff
@@ -130,3 +156,37 @@ Great! Marketplace content -- including visualizations -- were designed for cont
 
 **`yarn.lock`**: [Yarn](https://yarnpkg.com/) is a package manager alternative to npm. This file serves essentially the same purpose as `package-lock.json`, just for a different package management system.
 
+## Troubleshooting
+
+### Build fails with an error code of `ERR_OSSL_EVP_UNSUPPORTED`
+
+If you run into the following error when building using `yarn build`:
+
+```shell
+yarn run v1.22.19
+$ webpack
+node:internal/crypto/hash:71
+this[kHandle] = new _Hash(algorithm, xofLen);
+                ^
+
+Error: error:0308010C:digital envelope routines::unsupported
```

Comment:
I don't think the entire stack needs to be included. probably just the title, which indicates the error, and the solution below ("use the following command...") is sufficient.

Consider whether this could just instead be a note under the **Compile your code** section above.